### ISSUE TITLE
Implement archeologic data from zuid holland

### ISSUE THREAD

**Issue by breyten** [openstate/open-cultuur-data#113]
They have an OAI-PMH interface as [described here](http://archeologie.zuid-holland.nl/oaiinfo.htm)

**Comment by tomkunzler** [comment]
Maybe we should wait implementing this.  I have made contact with Provincie Zuid Holland. We are also receiving archeological data from Noord-Holland later this year (participants of the OCD Masterclass). Would be good to combine this in one effort and maybe in this way have a conversation about what we can do as OS for PZH.

**Comment by breyten** [comment]
Yeah we're aware. It won't be put live until we have them both ;) -- But it's likely that they'll have a different API anyway, so it's not all bad ;)
